ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -876,6 +876,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4
+- as: e2e-aws-ipi-hypershift-onprem-p1
```

Comment:
Workflow name `aws-ipi-ovn-hypershift` should be included in job name. So that we can easily identify which profile we are testing in RP.

Reply:
It is a new added test set for all HyperShift installation testcases. There is no existing related Jenkins job now.  He realized this test directly in prow.

Reply:
ping me if you are not sure what's the issue I'm talking about.

Reply:
@liangxia I already understand what you mean, I will update it

Reply:
Got it. @LiangquanLi930 will update the job name